corona-warn-app/cwa-app-android
# Sharing dialog error adjustment (EXPOSUREAPP-7116)

This PR addresses Exposureapp-7116.

**How to test:**
Steps:
App is installed
tap on "share error log"
switch off wifi
confirm and try to send the els

**Expected result:**
correct error message regarding the els sharing process
Text for the error message in ELS: "Der Fehlerbericht kann aktuell nicht gesendet werden. Bitte versuchen Sie es später noch einmal."

**before PR Result:**
"Die Befragung kann aktuell nicht aufgerufen werden. ..."
